ci/c0-azure-concourse-poc-params.yml

```diff
@@ -24,37 +24,43 @@ pcf_ssh_key_priv: |
 # ERT Cert & Key , if the string 'generate' is passed, pipeline will autogen certs
 pcf_ert_ssl_cert: |
 
-pcf_ert_ssl_key: |
+pcf_ert_ssl_key:
 
 ##############################################
 ### IaaS Specific Params
 ##############################################
 # Wipe Arg(s) anything other than wipe bypasses the wipe-env job
 arg_wipe: wipe
 ### azure ###
+azure_access_key:
+azure_account_name:
+azure_buildpacks_container:
+azure_droplets_container:
+azure_packages_container:
 azure_pcf_terraform_template: c0-azure-base # 'Template' for pipeline to determine Terrafrom & Opsman JSON config
-azure_terraform_prefix: c0-opsman-validation # Prefix tag for Terraform created Objects
-azure_subscription_id: "" # REQD
-azure_tenant_id: "" # REQD
+azure_region: eastus # valid Azure region reqd
+azure_resources_container:
 azure_service_principal_id: "" # REQD
 azure_service_principal_password: "c0P1v0tal"
-azure_region: eastus # valid Azure region reqd
-azure_terraform_vnet_cidr: "192.168.0.0/20"
-azure_terraform_subnet_infra_cidr: "192.168.0.0/26"
-azure_terraform_subnet_infra_reserved: "192.168.0.1-192.168.0.9"
-azure_terraform_subnet_infra_dns: "168.63.129.16,8.8.8.8"
-azure_terraform_subnet_infra_gateway: "192.168.0.1"
+azure_subscription_id: "" # REQD
+azure_tenant_id: "" # REQD
+azure_terraform_prefix: c0-opsman-validation # Prefix tag for Terraform created Objects
```

Comment:
This is not a good default; it should be a generic name, or even generated by Terraform as via [random_pet](https://www.terraform.io/docs/providers/random/r/pet.html).